test/utils/test_loop.py

```diff
@@ -32,6 +33,10 @@ def test_remove_self_loops():
     assert out[0].tolist() == expected
     assert out[1].tolist() == [[1, 2], [3, 4]]
 
+    adj = to_torch_coo_tensor(edge_index)
```

Comment:
`to_troch_coo_tensor` returns a sparse `torch.Tensor`. It would be nice to add a test for `SparseTensor` too.

Reply:
Do you mean to add tests with `torch_sparse.SparseTensor` as input for these functions as well?

Reply:
Actually skip that. I guess we only want to support sparse `torch.Tensor` here, right?

Reply:
Yes, exactly.